test/Microsoft.Health.Fhir.Ingest.UnitTests/Telemetry/FhirExceptionTelemetryProcessorTests.cs

```diff
@@ -30,7 +31,7 @@ public void GivenHandledExceptionTypes_WhenHandleExpection_ThenMetricLoggedAndTr
             var ex = Activator.CreateInstance(exType) as Exception;
 
             var exProcessor = new FhirExceptionTelemetryProcessor();
-            var handled = exProcessor.HandleException(ex, log);
+            var handled = exProcessor.HandleException(ex, new JArray(), log);
```

Comment:
For the Fhir Transformation was trying to include the measurement group as the payload logged with the error which I believe is an array. If we do a bit more work to get the individual measurement then I believe JObject would be more appropriate yes. Might end up just using a byte array though like you suggested to avoid the conversion